src/ORM/Table.php

```diff
@@ -243,6 +247,16 @@ public function __construct(array $config = [])
         if (!empty($config['associations'])) {
             $associations = $config['associations'];
         }
+        if (!empty($config['validator'])) {
+            if (!is_array($config['validator'])) {
+                $this->validator(null, $config['validator']);
```

Comment:
Yeah, we caught that just after the comment. We got PWND in our test as calling the "default" validation set *does* actually return a Validator, just not the one we expected to get.

I did suggest though that in referencing the value in various places it might make sense to store it in a constant.